apps/avatax/vitest.workspace.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-default-export */
+import { defineWorkspace } from "vitest/config";
+
+export default defineWorkspace([
+  {
+    extends: "./vitest.config.ts",
+    test: {
+      name: "react",
+      exclude: ["integration/**/*.spec.ts"],
+    },
+  },
+  {
+    test: {
+      include: ["integration/**/*.spec.ts"],
+      setupFiles: ["./integration/setup.ts"],
+      name: "integration",
+      environment: "node",
+      testTimeout: 20_000, // sync webhook timeout is 20s
```

Comment:
but not only sync webhooks exist. Its fine to have it as a default, but make a comment more descriptive (that this is default, because of sync, but can be changed in each test suite)